pkg/e2e/.gitignore

```diff
@@ -0,0 +1 @@
+_cache
```

Comment:
I don't think this exists in the current version so this whole file is redundant.

Reply:
Indeed